packages/qwik/src/optimizer/src/plugins/vite-error.ts

```diff
@@ -0,0 +1,210 @@
+import type { RollupError } from 'rollup';
+import type { ViteDevServer } from 'vite';
+import { yellow, magenta, cyan } from 'kleur/colors';
+import strip from 'strip-ansi';
+
+/**
```

Comment:
i think about creating another PR on the vite repo to have an option to pass in some custom markdown into the vite error handler so we can remove the whole logic again on our side.